sputnikdao/src/lib.rs

```diff
@@ -312,6 +345,10 @@ impl SputnikDAO {
         self.purpose.clone()
     }
 
+    pub fn get_mode(&self) -> ProposalMode {
```

Comment:
looks orphan?

Reply:
that a public view function so one can query the mode of the contract.
Actually thinking to consolidate all settings into a single view call to reduce number of RPC calls